### PR TITLE
fix(widget-chat): improve delete functionality 

### PR DESCRIPTION
**Pull request by adamweeks**

### CODE REVIEW

File: packages/widget-chat/src/components/icon/index.js [hunk 1]
```diff
@@ -4,14 +4,14 @@ import classNames from 'classnames';
...
 import styles from './styles.css';
 
-export const ICON_TYPE_ADD = `add`;
-export const ICON_TYPE_DELETE = `delete`;
-export const ICON_TYPE_MESSAGE = `message`;
-export const ICON_TYPE_RIGHT_ARROW = `rightArrow`;
+export const ICON_TYPE_ADD = `icon-cisco-add`;
+export const ICON_TYPE_DELETE = `icon-cisco-exit-outline`;
+export const ICON_TYPE_MESSAGE = `icon-cisco-message`;
+export const ICON_TYPE_RIGHT_ARROW = `icon-cisco-right-arrow`;
```
Comment (adamweeks) [inline packages/widget-chat/src/components/icon/index.js]: Tons of icons added in the css, but only want to list the ones we use for now.

**Comment by ianwremmel** [comment]
reminder: commit descriptions should be statements. the description here makes it sound like you're removing functionality improvements :)

In the near future, these commit statements will be used to automate changelog generation (and possibly major/minor/patch version numbers).

**Comment by adamweeks** [comment]
@ianwremmel updated PR name: fix(widget-chat): improve delete functionality

File: packages/widget-chat/src/components/confirmation-modal/styles.css [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+  font-weight: 300;
+  line-height: 1.6;
+  color: var(--color-gray);
+  text-align: center;
+}
+
+.dialogue-modal-body-subtext {
+  height: 100%;
+  margin: 16px 24px 32px;
+  font-family: CiscoSansTTExtraLight, 'Helvetica Neue', Arial;
```
Comment (bzang) [inline packages/widget-chat/src/components/confirmation-modal/styles.css]: Is the `CiscoSansTTExtraLight` being picked up? I think it's supposed to be `'CiscoSans TT Extra Light'`.

Comment (bzang) [inline packages/widget-chat/src/components/confirmation-modal/styles.css]: See above references as well

Comment (adamweeks) [inline packages/widget-chat/src/components/confirmation-modal/styles.css]: Oh, bah. I got this from the web client and forgot we changed the name to have spaces. It probably is picking up the default style. I'll fix it

**Pull request merged**